src/amo/addonManager.js

```diff
@@ -116,7 +120,7 @@ export function hasAbuseReportPanelEnabled(
 export function reportAbuse(
   addonId: string,
   { _mozAddonManager = window.navigator.mozAddonManager }: OptionalParams = {},
-) {
+): void | Promise<mixed> | Promise<boolean> {
```

Comment:
how can it be `void`?

Reply:
It should be `Promise<type of the reportAbuse response>`

Reply:
The type of the response is documented above as `boolean`, so I guess that should suffice.